### PR TITLE
add first version of check_word

### PR DESCRIPTION
**Pull request by lasconic**
- [ ] Fixes #
- [ ] Tests added/updated.
- [x] `./check.sh` run and eventual issues fixed.
- [x] Updated CONTRIBUTORS.md, if not already done in the past.

### CODE REVIEW

**Comment by lasconic** [comment]
Just a quick test but it's already helpful. Tested with 

```
python -m wikidict fr --check-word "viandaille"
```
No output !!

tested with "être"

```
python -m wikidict fr --check-word "être"
```
Lot of output...

**Comment by BoboTiG** [comment]
I improved the script to display a colored diff:
```diff
diff --git a/wikidict/check_word.py b/wikidict/check_word.py
index 54d8c7a..bc08ec9 100644
--- a/wikidict/check_word.py
+++ b/wikidict/check_word.py
@@ -30,14 +30,15 @@ def main(locale: str, word: str) -> int:
     if details.etymology:
         clean_text = BeautifulSoup(details.etymology, features="html.parser").text
         if clean_text not in text:
-            print("Etymology KO")
-            print(clean_text)
-
-    for definition in details.definitions:
-        if definition:
-            clean_text = BeautifulSoup(definition, features="html.parser").text
-            if clean_text not in text:
-                print("definition KO")
-                print(clean_text)
+            print("Etymology")
+            print("\033[31m-", details.etymology, "\033[0m")
+            print("\033[32m+", clean_text, "\033[0m")
+
+    for num, definition in enumerate(details.definitions):
+        clean_text = BeautifulSoup(definition, features="html.parser").text
+        if clean_text not in text:
+            print(f"\nDefinition n°{num}")
+            print("\033[31m-", definition, "\033[0m")
+            print("\033[32m+", clean_text, "\033[0m")
 
     return 0
```

And it seems the comparison is not good because `clean_text` contains stripped HTML, but not `details.*`.

File: wikidict/check_word.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+from .stubs import Word
+
+
+def get_word(word: str, locale: str) -> Word:
+    """Get a *word* wikicode and parse it."""
+    url = f"https://{locale}.wiktionary.org/w/index.php?title={word}&action=raw"
+    with requests.get(url) as req:
+        code = req.text
+
+    return parse_word(word, code, locale, force=True)
```
Comment (BoboTiG) [inline wikidict/check_word.py:15]: I guess `force=True` could be remove.

**Comment by BoboTiG** [comment]
Thanks, I pushed and will work on it locally.

**Comment by lasconic** [comment]
OK. Your diff is not good though. We search clean_text in text  and both are full text, no HTML tags.
It's hard to make a diff since we don't know where the different is.
We could do a fuzzy search or search for some part of clean text only, or for a start : do a second search with all spaces (breakable or not) converted to normal spaces etc... to find where clean_text is most likely.

**Comment by BoboTiG** [comment]
Yup, I got it too late :)
Fuzzy search would be a killer feature though.

BTW I just pushed some changes. And we have some work incoming haha

**Comment by BoboTiG** [comment]
I noticed that there are double spaces around "notes", maybe the check could remove all spaces to workaround such issue. It would be a good first move. I will try it.

**Pull request merged**